### PR TITLE
ref: remove play button by default

### PR DESCRIPTION
**Pull request by joshuarli**
This is the beginning of some small UI mods I wanna make, so there's some other misc change in here.

After this we only have the Play+ button which is a much better intentional default.

image

### CODE REVIEW

File: buildSrc/src/main/groovy/com/thoughtworks/go/build/docker/BuildDockerImageTask.groovy [hunk 1]
```diff
@@ -111,15 +111,6 @@ class BuildDockerImageTask extends DefaultTask {
         verifyHelper.call(isNativeVerify)
         logger.lifecycle("\nVerification of ${imageNameWithTag} image on ${distro.dockerVerifyArchitecture} successful.")
       }
-
-      logger.lifecycle("Cleaning up...")
-      // delete the image, to save space
-      if (!project.hasProperty('dockerBuildKeepImages')) {
```
Comment (mattgauntseo-sentry) [inline buildSrc/src/main/groovy/com/thoughtworks/go/build/docker/BuildDockerImageTask.groovy:117]: Why did you need to remove this?

**Review by mattgauntseo-sentry — APPROVED** [review]

File: buildSrc/src/main/groovy/com/thoughtworks/go/build/docker/BuildDockerImageTask.groovy (hunk 1 above)
Comment (joshuarli) [reply buildSrc/src/main/groovy/com/thoughtworks/go/build/docker/BuildDockerImageTask.groovy:117]: Because it was deleting the image that was being built. Even after passing a self check.
